README.md

```diff
@@ -208,6 +208,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [Chatwoot](https://www.chatwoot.com) - Self-hosted customer communication platform, an alternative to Intercom & Zendesk. ([Source Code](https://github.com/chatwoot/chatwoot)) `MIT` `Ruby`
 - [Cherry](https://github.com/rafael-santiago/cherry) - Tiny webchat server. `GPL-2.0` `Go`
 - [Conduit](https://conduit.rs/) - A simple, fast, and reliable chat server powered by Matrix. ([Source Code](https://gitlab.com/famedly/conduit)) `Apache-2.0` `Rust`
+- [CoreDB](https://github.com/rolandosborne/CoreDB) - A decentralized network node for hosting your digital identity, supports the mobile app, IndiView, for sharing photos, videos and chatting. `Apache-2.0` `Javascript` `Java`
```

Comment:
Is there a specific reason to use your fork over the official repo?

Reply:
Rey

Reply:
> Is there a specific reason to use your fork over the official repo?

Reply:
Hi, I am the dev for the original project, but we will be closing it due to lack of traction and non-profit fees. I wanted to show the history of the project, so I forked it.

Reply:
Ah, now you have rewritten the git history and deleted the old orga. I personally would have made it the other way around, keep the github orga (after all they are free, so no need for a non-profit) and operate from there.

Reply:
I was worried the non-profit shutting down and the decentralized network being part of a separate GitHub would be confusing. You're probably right though, I should have left it alone.